src/index.js

```diff
@@ -0,0 +1,23 @@
+import * as jestPuppeteerConfig from './jest-puppeteer-config';
+import * as pageUtils from './page-utils';
+import * as setup from './setup';
+import { jestConfig } from './jestConfig';
+import Page from './page';
+
+exports.jestPuppeteerConfig = jestPuppeteerConfig;
```

Comment:
I think we mixing ES6 and CommonJS here. Is this intentional?

Reply:
No, not intentional. updated.